code/pizzabot.js

```diff
@@ -1,7 +1,40 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const A = "Vegetarian Pizza"
+const B = "Hawaiian Pizza"
+const C = "Pepperoni Pizza"
```

Comment:
In other parts in this file you are using `'` instead of `"` so it would be great if your file follow the same rules!
```suggestion
const C = 'Pepperoni Pizza'
```